tests/plugins/ml/ml/tasks.yaml

```diff
@@ -0,0 +1,32 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test the creation of tasks.
+distributions:
+  excluded:
+    - amazon-managed
+    - amazon-serverless
```

Comment:
Do you know for sure this API is not available in AOS? If not you can remove this for now.